.env.example

```diff
@@ -30,3 +30,7 @@ SMTP_PORT=1025
 HOSTNAME=http://host.docker.internal:8080
 # The SimpleSAML IDP's XML metadata
 SAML_IDP_METADATA_PATH=config/idp-metadata-dev.xml
+
+# Background jobs
+EXPIRATION_REPEAT_FREQUENCY=86400000
```

Comment:
What you can do here is leave a comment that explains what the humber means:

```
# 24 * 7 * ... = 86400000
EXPIRATION_REPEAT_FREQUENCY=86400000
```

Best of both worlds.

Reply:
Also, when you have a number like this, it's good to name it with units: `EXPIRATION_REPEAT_FREQUENCY_SECONDS` or `_MS` etc

Reply:
Agreed. It's helpful to know what the numbers mean.